modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/dto/v1_0/converter/ObjectEntryDTOConverter.java

```diff
@@ -230,7 +231,21 @@ public ObjectEntry toDTO(
 		ObjectDefinition objectDefinition = _getObjectDefinition(
 			dtoConverterContext, serviceBuilderObjectEntry);
 
-		ObjectEntry objectEntry = new ObjectEntry() {
+		ObjectEntryVersion objectEntryVersion =
+			(ObjectEntryVersion)dtoConverterContext.getAttribute(
+				"objectEntryVersion");
+
+		ObjectEntry contentObjectEntry;
+
+		if (objectEntryVersion != null) {
+			contentObjectEntry = ObjectEntry.unsafeToDTO(
+				objectEntryVersion.getContent());
+		}
+		else {
+			contentObjectEntry = null;
+		}
```

Comment:
@mxs2 initialize contentObjectEntry as null, this way you will not need to add the `else` clause:

````java
ObjectEntry contentObjectEntry = null;

if (objectEntryVersion != null) {
	contentObjectEntry = ObjectEntry.unsafeToDTO(
		objectEntryVersion.getContent());
}
````